### PR TITLE
GG-7031 added open api spec files

### PR DESCRIPTION
**Pull request by jinbal-hmrc**

### CODE REVIEW

File: public/api/conf/1.0/common/overview.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The User Information API provides information about the signed-in user, in an OpenID Connect compliant format.
```
Comment (ModisR) [inline public/api/conf/1.0/common/overview.md:1]: The directory `/common` should be directly under `/public/api/conf` as opposed to the version-specific directories:
https://confluence.tools.tax.service.gov.uk/display/DTRG/RAML+to+OpenAPI+Specification+%28OAS%29+migration+guide#RAMLtoOpenAPISpecification(OAS)migrationguide-APIOverviewandCommonSections

File: public/api/conf/1.1/common/overview.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The User Information API provides information about the signed-in user, in an OpenID Connect compliant format.
```
Comment (ModisR) [inline public/api/conf/1.1/common/overview.md:1]: Same applies as for `1.0`.

File: public/api/conf/1.0/application.yaml [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+    * 'email': email
+    * 'openid:hmrc-enrolments': hmrc_enrolments
+    * 'openid:government-gateway': government_gateway
+    * 'openid:mdtp': mdtp
+    * 'openid:gov-uk-identifiers': uk_gov_nino
+  contact: {}
+  version: '1.0'
+servers:
+- url: https://api.service.hmrc.gov.uk/
+  variables: {}
```
Comment (ModisR) [inline public/api/conf/1.0/application.yaml:18]: At root level, we need to insert a `securitySchemes` property as per this example: <https://github.com/hmrc/open-api-example-microservice/blob/main/resources/public/api/conf/1.0/application.yaml#L42>.

Ensure the scopes match those provided in the API definition: https://github.com/hmrc/openid-connect-userinfo/blob/main/app/views/definition.scala.txt.

Finally, each endpoint needs a `security` property declaring to which scopes it is restricted, as shown on these lines:
- https://github.com/hmrc/open-api-example-microservice/blob/main/resources/public/api/conf/1.0/application.yaml#L91
- https://github.com/hmrc/open-api-example-microservice/blob/main/resources/public/api/conf/1.0/application.yaml#L137
- https://github.com/hmrc/open-api-example-microservice/blob/main/resources/public/api/conf/1.0/application.yaml#L184

You may need to refer to the RAML or consult someone familiar with this service to determine what those restrictions are.

File: public/api/conf/1.0/docs/scopes.md [hunk 4]
```diff
@@ -2,7 +2,7 @@ Access to user information is controlled through scopes. Each access token (OAut
```
Comment (ModisR) [inline public/api/conf/1.0/docs/scopes.md:2]: Scopes.md can be deleted. All its content is included in the OAS YAML `info.description` property.

**Review by ModisR — CHANGES_REQUESTED** [review]
Just 1 more change (to Scopes.md) needed from what I see.

My understanding of security schemes is a little shaky, TBH, but your changes look good and in my experience the pipeline will tell you if you went wrong somewhere. Ultimately, we can always rely on Team API to answer any questions. :)

File: public/api/conf/1.0/docs/scopes.md (hunk 4 above)
Comment (jinbal-hmrc) [reply public/api/conf/1.0/docs/scopes.md:2]: was planning to delete all the legacy docs/files in a separate PR once its been validated

**Review by ModisR — APPROVED** [review]

**Pull request merged**